css/main.css

```diff
@@ -5,11 +5,14 @@
 /*************************
 *******Typography******
 **************************/
+* {
+  border: px solid red;
+}
 
 body {
   font-family: 'Dosis', sans-serif;
   font-size: 17px;
-  color: #fff
+  color: #fff;
```

Comment:
Just a question "Would anything break if i don't put semi-colon here?"
If no, then why did you put that in first place and if yes, then why?